src/wazuh_db/wdb_parser.c

```diff
@@ -652,7 +652,7 @@ int wdb_parse(char * input, char * output, int peer) {
 
         mdebug2("Global query: %s", query);
 
-        if (wdb = wdb_open_global(), !wdb) {
+        if (wdb = wdb_open_global(), !wdb || !wdb->enabled) {
```

Comment:
Shouldn´t we allow only the backup command to be executed when the DB is disabled? If this is the case, we should re-set it to enabled after the restore.

Reply:
The database now is set to enabled when the backup is restored.

Reply:
About your comment, we decided to directly reject all the calls to `global.db` as the API endpoints for the backup management will not be implemented by the moment.